lte/gateway/c/core/oai/tasks/nas5g/src/M5GAuthenticationReject.cpp

```diff
@@ -62,35 +58,31 @@
 }
 
 // Encoding Authentication Reject Message and its IEs
-int AuthenticationRejectMsg::EncodeAuthenticationRejectMsg(
-    AuthenticationRejectMsg* auth_reject, uint8_t* buffer, uint32_t len) {
+int AuthenticationRejectMsg::EncodeAuthenticationRejectMsg(uint8_t* buffer,
+                                                           uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.